start_pokealarm.py

```diff
@@ -285,6 +296,42 @@ def download_data(sigdiff=None):
         # All's done! Overwrite the old local file data
         os.replace("data/tmp_charged_moves.json", "data/charged_moves.json")
 
+    if sigdiff is None or sigdiff['shiny_possible']:
+        log.info("New shiny_possible data found! Fetching in progress...")
+
+        # Fetch data
+        shiny_possible_url = "https://raw.githubusercontent.com/jms412/" \
+            "PkmnShinyMap/main/shinyPossible.json"
+        shiny_possible_data = requests.get(shiny_possible_url).json()
+
+        # Check some dict paths which don't have to change
+        if shiny_possible_data["map"]["1"] != " \u2728":
```

Comment:
Poracle actually only cares that it is present and non-empty. In the old days the map was used to give us the symbol but no longer.  I am sure this will be preserved so just an FYI.

Reply:
That's just some security checks we use to do for each remotely file ^^